configurations/manager/6TB_dataset.yaml

```diff
@@ -0,0 +1,32 @@
+test_duration: 1200
+round_robin: true
+
+prepare_write_cmd: [ "cassandra-stress write cl=ALL n=1500000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=650 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..1500000000",
+                     "cassandra-stress write cl=ALL n=1500000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=650 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1500000001..3000000000",
+                     "cassandra-stress write cl=ALL n=1500000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=650 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=3000000001..4500000000",
+                     "cassandra-stress write cl=ALL n=1500000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=650 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=4500000001..6000000000" ]
+
+stress_cmd: [ "cassandra-stress write cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=500 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..3000000000",
+              "cassandra-stress write cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=500 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=3000000001..6000000000", ]
+
+stress_read_cmd: [ "cassandra-stress read cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=500 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..3000000000",
+                   "cassandra-stress read cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=500 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=3000000001..6000000000", ]
+
+instance_type_db: 'i4i.8xlarge'
+instance_type_loader: 'c6i.8xlarge'
+n_db_nodes: 6
+n_loaders: 4
+n_monitor_nodes: 1
+region_name: us-east-1
+backup_bucket_location: 'manager-backup-tests-us-east-1'
+append_scylla_yaml:
+  auto_snapshot: false
+  object_storage_endpoints:
+    - name: s3.us-east-1.amazonaws.com
+      port: 443
+      https: true
+      aws_region: us-east-1
+      iam_role_arn: arn:aws:iam::797456418907:instance-profile/qa-scylla-manager-backup-instance-profile
```

Comment:
this should never be part of a public repository in my opinion, not sure how to deal with this in a better way. Do we handle secrets in the SCT? I believe we are fetching some stuff from the AWS right?

Reply:
i guess this might be an alternative:
```
-      iam_role_arn: arn:aws:iam::797456418907:instance-profile/qa-scylla-manager-backup-instance-profile
-
+aws_instance_profile_name_db: 'qa-scylla-manager-backup-instance-profile'
```
i'll give it a test.

Reply:
passed ok.

Reply:
the role should come from the instance profile, we don't expect user to put those in Scylla configuration.

if it's not yet working, it's should be a scylla core bug (as I told the people developing this feature, multiple times)

Reply:
@fruch - please create an issue describing your expectation, please assign it to me.

Reply:
I'm not opening issue for that, 

AWS credentials should work exactly as in KMS, 
i.e. it should support getting credentials from instance profile
https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html